test/e2e/apps/cronjob.go

```diff
@@ -403,7 +409,9 @@ var _ = SIGDescribe("CronJob", func() {
 			[]byte(`{"metadata":{"annotations":{"patchedstatus":"true"}},"status":`+string(cjStatusJSON)+`}`),
 			metav1.PatchOptions{}, "status")
 		framework.ExpectNoError(err)
-		framework.ExpectEqual(patchedStatus.Status.LastScheduleTime.Equal(&now1), true, "patched object should have the applied lastScheduleTime status")
+		if !patchedStatus.Status.LastScheduleTime.Equal(&now1) {
+			framework.Fail("patched object should have the applied lastScheduleTime status")
```

Comment:
Would it make sense to add the expected and actual time here, as in the next check?

Reply:
Changed 👍🏻 

Much better in case of error.